user_guide_src/source/database/results.rst

```diff
@@ -283,6 +283,19 @@ Make sure to call the method using your query result object::
 
 	echo $query->getFieldNames();
 
+**getNumRows()**
+
+The number of records returned by the query. Make sure to call
+the method using your query result object::
+
+	$query = $db->query('SELECT * FROM my_table');
+
+	echo $query->getNumRows();
+
+.. note:: Because PHP's SQLite3 driver lacks an efficient method returning a record
```

Comment:
It sounds to me like it's a PHP fault, but it isn't.
```suggestion
.. note:: Because SQLite3 lacks an efficient method returning a record
```

Reply:
change made.